kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -117,6 +117,12 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     context:
       "\nCould also be translated as \"View information about providing identifier\"\n\nAll 'AriaLabel' type of messages are providing additional context to the screen-reader users. \n\nIn this case the screen-reader will announce the message to the user indicating that they can access more information and examples about the 'Identifier' through the 'i' icon.",
   },
+
+  // Content activity
+  notStartedLabel: {
+    message: 'Not started',
+    context: 'For content that has not been viewed or engaged with yet',
```

Comment:
I know this is context and not the user string, but still remembering that we should steer away from the term `content` :slightly_smiling_face: 

Maybe `Used to indicate a resource that learners did not yet open and engage with`?

Reply:
Is "resource" the most generic term? I would probably even pick something more generic than "content" because "Not started" is a status for things like Exams and Lessons.

Reply:
In the coach tools we actually have distinct labels for each type to ensure that the gender agreement is correct. For example:

Lesson not started:

https://github.com/learningequality/kolibri/blob/4cf820b14386aecc228fecff64c847bad407cbb1/kolibri/plugins/coach/assets/src/views/common/status/statusStrings.js#L122

Exercise not started:

https://github.com/learningequality/kolibri/blob/4cf820b14386aecc228fecff64c847bad407cbb1/kolibri/plugins/coach/assets/src/views/common/status/statusStrings.js#L66

Also for:

* Question
* Quiz
* Resource